src/other/mlpmixer.jl

```diff
@@ -1,83 +1,106 @@
-# Utility function for creating a residual block with LayerNorm before the residual connection
-_residualprenorm(planes, fn) = SkipConnection(Chain(fn, LayerNorm(planes)), +)
+"""
+    mixerblock(planes, npatches; mlp_ratio = (0.5, 4.0), mlp_layer = mlp_block, 
+               dropout = 0., drop_path_rate = 0., activation = gelu)
 
-# Utility function for 1D convolution
-_conv1d(inplanes, outplanes, activation = identity) = Conv((1, ), inplanes => outplanes, activation)
+Creates a feedforward block for the MLPMixer architecture.
+([reference](https://arxiv.org/pdf/2105.01601))
 
+# Arguments:
+- `planes`: the number of planes in the block
+- `npatches`: the number of patches of the input
+- `mlp_ratio`: number(s) that determine(s) the number of hidden channels in the token mixing MLP 
+               and/or the channel mixing MLP as a ratio to the number of planes in the block.
+- `mlp_layer`: the MLP layer to use in the block
+- `dropout`: the dropout rate to use in the MLP blocks
+- `drop_path_rate`: Stochastic depth rate
+- `activation`: the activation function to use in the MLP blocks
 """
-    mlpmixer(imsize::NTuple{2} = (256, 256); inchannels = 3, patch_size = 16, planes = 512, 
-             depth = 12, expansion_factor = 4, dropout = 0., nclasses = 1000, token_mix = 
-             _conv1d, channel_mix = Dense))
+function mixerblock(planes, npatches; mlp_ratio = (0.5, 4.0), mlp_layer = mlp_block, 
+                    dropout = 0., drop_path_rate = 0., activation = gelu)
+  tokenplanes, channelplanes = [Int(r * planes) for r in mlp_ratio]
+  return Chain(SkipConnection(Chain(LayerNorm(planes),
+                                    x -> permutedims(x, (2, 1, 3)),
```

Comment:
```suggestion
                                    Base.Fix2(permutedims, (2, 1, 3)),
```
Or a separate function since this operation seems to show up in a few places.